locales-pl-PL.xml

```diff
@@ -211,50 +214,50 @@
     <term name="long-ordinal-09">dziewiąty</term>
     <term name="long-ordinal-10">dziesiąty</term>
 
-    <!-- LONG LOCATOR FORMS -->
-    <term name="act">			 
+    <!-- LONG LOCATOR FORMS --> <!--K.K. I would aim for 100% translation, but some are problematic -->
+    <term name="act"> <!--K.K. This one should be left untranslated and for the style creator to decide imo, as depending on the context translation can vary (for example in law the plural form would be akta, but in play it would be akty)-->
       <single>act</single>
-      <multiple>acts</multiple>						 
+      <multiple>acts</multiple>
     </term>
-    <term name="appendix">			 
-      <single>appendix</single>
-      <multiple>appendices</multiple>						 
+    <term name="appendix"> <!--K.K. see: https://sjp.pwn.pl/poradnia/haslo/Aneks-a-apendyks;22723.html-->
+      <single>załącznik</single>
+      <multiple>załączniki</multiple>						 
     </term>
-    <term name="article-locator">			 
-      <single>article</single>
-      <multiple>articles</multiple>						 
+    <term name="article-locator"> <!--K.K.-->
+      <single>artykuł</single>
+      <multiple>artykuły</multiple>						 
     </term>
-    <term name="canon">			 
+    <term name="canon"> <!--K.K. not sure what that means -->
```

Comment:
"Canon" is a sub-part of some classical works, similar to "book". Looks like "kanon"/"kan." might be the right translation

https://www.ekai.pl/wp-content/uploads/2021/12/KodeksPrawaKanonicznego_2021_12_15_pl.pdf